tasks/ghost-os.yml

```diff
@@ -4,3 +4,9 @@
   template: src=etc/init/ghost.conf.j2 dest=/etc/init/ghost.conf
   notify: restart ghost
   tags: [ghost]
```

Comment:
I think It would be relatively easy to check if the system is using upstart/systemd when Ansible is gathering facts, I think we could add that condition in the future, but right now I wouldn't bother, if upstart/systemd are running both would work now, and if upstart file is there for 15.04 won't matter because the service running will get notified through systemd.